playbooks/templates/network_throughput.yaml.j2

```diff
@@ -1,6 +1,6 @@
 {% set label = "network_throughput_"+item.0.name %}
 {% set check_name = label+'--'+ansible_hostname %}
-{% set max_speed = (item.0.max_speed | default(0, boolean=true) or item.1.stdout) | int * 10 ** 6 / 8 %}
```

Comment:
What's with the item1.stdout thing?  seems like a logic change to me at first glance.